apps/user/src/types/searchMap.type.ts

```diff
@@ -23,10 +23,18 @@ export interface StoreResponseData {
   storeList: Store[];
 }
 
+// 성공 응답: success + data
 export interface StoreApiSuccessResponse {
```

Comment:
타입이 store로 되어 있어서 혼란스러울 수 있을것 같습니다!
searchMap이 명시적으로 들어가면 좋지 않을까 싶습니다!

그리고 이 타입들은
`apps/user/src/apis/types/index.ts` 여기에 정의되어 있는 공통 api 사용해도 될거같아요
```suggestion
export type SearchMapSuccessResponse = ResultSuccess<SearchMapResponseData>
```